ansible/roles/box/prepare/templates/Vagrantfile.j2

```diff
@@ -6,7 +6,7 @@ Vagrant.configure(2) do |config|
 
     config.vm.provider "libvirt" do |domain, override|
         override.vm.box_url = "{{ base_box_url }}"
-        override.vm.box = "{{ base_box_name }}"
+        override.vm.box = "f{{ fedora_version }}"
```

Comment:
Don't we have a typo here? Shouldn't be `"{{ fedora_version }}"` (without the `f')?